contracts/multisig-prover/src/state.rs

```diff
@@ -26,6 +30,7 @@ pub struct Config {
     pub worker_set_diff_threshold: u32,
     pub encoder: Encoder,
     pub key_type: KeyType,
+    pub domain_separator: Hash,
```

Comment:
Are we planning to migrate the old contract to this contract, or are we just redeploying?